src/QMCHamiltonians/ACForce.cpp

```diff
@@ -47,21 +49,27 @@ std::unique_ptr<OperatorBase> ACForce::makeClone(ParticleSet& qp, TrialWaveFunct
 std::unique_ptr<OperatorBase> ACForce::makeClone(ParticleSet& qp, TrialWaveFunction& psi_in, QMCHamiltonian& ham_in)
 {
   std::unique_ptr<ACForce> myclone = std::make_unique<ACForce>(ions_, qp, psi_in, ham_in);
+  myclone->fastDerivatives_=fastDerivatives_;
+  myclone->useSpaceWarp_=useSpaceWarp_;
+  myclone->first_force_index_=first_force_index_;
   return myclone;
 }
 
 bool ACForce::put(xmlNodePtr cur)
 {
-  std::string useSpaceWarpString("no");
   std::string ionionforce("yes");
   RealType swpow(4);
   OhmmsAttributeSet attr;
-  attr.add(useSpaceWarpString, "spacewarp"); //"yes" or "no"
+  attr.add(useSpaceWarp_, "spacewarp",{false}); //"yes" or "no"
```

Comment:
Do you have clang-format around? If yes, please format touched files.

Reply:
Done.